frontend/src/typeHelpers.ts

```diff
@@ -93,3 +93,8 @@ export type EitherNotBoth<TypeA, TypeB> = (TypeA & Never<TypeB>) | (TypeB & Neve
 export type EitherOrNone<TypeA, TypeB> =
   | EitherNotBoth<TypeA, TypeB>
   | (Never<TypeA> & Never<TypeB>);
+
+export const isSomeEnum =
```

Comment:
Um.... Why? Help me understand why we need to dynamically check a static state of if it's a TypeScript enum?

Reply:
Oooooh, I misread the use -- you're checking if it's IN the enum... this is a bad named function. `isInEnum` is probably better. Currying functions might have some other naming strategy, but I think that's better than "is this an enum" sounding function.